Subject: Set perl installation path to vendorlib
Author: Tzafrir Cohen <tzafrir.cohen@xorcom.com>
Forwarded: no

Point Zaptel PERL modules to the right directory under which to install
perl modules.

(sitelib in Debian is under /usr/local. We don't want to install there)

Debian-specific configuration.

--- a/xpp/Makefile
+++ b/xpp/Makefile
@@ -28,7 +28,7 @@ SBINDIR		= $(prefix)/sbin
 DATADIR		= $(datadir)/dahdi
 MANDIR		= $(mandir)/man8
 HOTPLUG_USB_DIR	= $(sysconfdir)/hotplug/usb
-PERLLIBDIR	:= $(shell eval `perl -V:sitelib`; echo "$$sitelib")
+PERLLIBDIR	:= $(shell eval `perl -V:vendorlib`; echo "$$vendorlib")
 PERL_DIRS	:= $(shell cd perl_modules; find * -name '[A-Z]*' -type d| xargs)
 PERL_MODS_PAT	:= *.pm $(PERL_DIRS:%=%/*.pm)
 PERL_MODS	:= $(shell cd perl_modules; echo $(PERL_MODS_PAT))
